packages/compass-e2e-tests/tests/instance-databases-tab.test.ts

```diff
@@ -135,15 +137,14 @@ describe('Instance databases tab', function () {
   });
 
   it('can refresh the list of databases using refresh controls', async function () {
-    const db = 'my-instance-database';
-    const coll = 'my-collection';
+    const dbName = 'my-instance-database';
+    const collName = 'my-collection';
     // Create the database and refresh
-    await browser.shellEval(`use ${db};`);
-    await browser.shellEval(`db.createCollection('${coll}');`);
     await browser.navigateToInstanceTab('Databases');
+    await createBlankCollection(dbName, collName);
```

Comment:
I felt a bit safe adding this because I was also cleaning up the database in the test itself but I can understand that the availability of such method does promote creating arbitrary databases.

Reply:
Your cleanup won't run if the test fails. I tend to prefer putting cleanups in before hooks over putting them inside tests or in after hooks for that reason.

Reply:
That sounds about right. I am gonna make the change as soon I get some perspective from the conversation below.